backends/tc/backend.cpp

```diff
@@ -910,8 +911,9 @@ safe_vector<const IR::TCKey *> ConvertToBackendIR::processCounterControlPathKeys
             auto temp_keys = HandleTypeNameStructField(field, extn, decl, kId, annoName);
             keys.insert(keys.end(), temp_keys.begin(), temp_keys.end());
         } else {
-            IR::TCKey *key =
-                new IR::TCKey(kId++, field->type->width_bits(), field->toString(), annoName, true);
+            cstring ptype = absl::StrCat("bit", std::to_string(field->type->width_bits()));
```

Comment:
You do not need `to_string`. Consider reading https://abseil.io/docs/cpp/guides/strings#abslstrcat-and-abslstrappend-for-string-concatenation